src/mlbstandings/web.py

```
@@ -1,17 +1,27 @@
 import backoff
 import requests
-
+# TODO remove once we're running on a system that works with urllib3 + baseball-reference.com
+import urllib.request # import Request, urlopen
+import urllib.error
+ 
```

Flake8 (7.2.0)

[warning] blank line contains whitespace

(W293)